OutSystems/outsystems-maps
# ROU-3579: Fixing marker on click

This PR fixes a bug where marker click event wasn't receiving correct coordinates.


### What was happening
* This was a side effect introduced in #104. We changed the coordinates parameter order.

### What was done
* Adjusted the parameter order.

### Test Steps
1. Click anywhere in the map.
Expected: A marker should be created on the place you clicked.

1. Open console
2. Press current screen button

Expected: No error should appear on console.
